### PR TITLE
feat: fetch prices from Aquarius in AssetList component

### PR DESCRIPTION
**Pull request by LucaMilanese90**
## Proposed Changes

  - fetch prices from Aquarius instead of Provider for the asset list view

### CODE REVIEW

**Comment by moritzkirstein** [comment]
We have lost a few features / designs in asset lists:
grafik

- `Dataset` / `Algorithm` chip
- Name of publisher should be loaded from credential
- Network below price

File: src/@utils/aquarius/index.ts [hunk 1]
```diff
@@ -291,24 +289,17 @@ export async function getPublishedAssets(
...
 
   const filters: FilterTerm[] = []
 
   filters.push(getFilterTerm('nft.state', [0, 4, 5]))
   filters.push(getFilterTerm('nft.owner', accountId.toLowerCase()))
-  accessType && filters.push(getFilterTerm('services.type', accessType))
+  accesType !== undefined &&
+    filters.push(getFilterTerm('services.type', accesType))
   type !== undefined && filters.push(getFilterTerm('metadata.type', type))
-  complianceType &&
```
Comment (nklomp) [inline src/@utils/aquarius/index.ts:305]: This is removing an additional filter present in FMDM

File: src/@utils/aquarius/index.ts [hunk 2]
```diff
@@ -291,24 +289,17 @@ export async function getPublishedAssets(
   ignoreState = false,
   page?: number,
   type?: string,
-  accessType?: string,
-  complianceType?: string
```
Comment (nklomp) [inline src/@utils/aquarius/index.ts:295]: This is a custom filter in FMDM

File: src/@utils/aquarius/index.ts (hunk 1 above)
Comment (LucaMilanese90) [reply src/@utils/aquarius/index.ts:305]: Sorry for that, I'll push a fix

**Review by moritzkirstein — APPROVED** [review]

**Comment by moritzkirstein** [comment]
Waiting for approval from @nklomp or Sphereon team before merging

**Review by nklomp — APPROVED** [review]
LGTM

**Pull request merged**